README.md

```diff
@@ -0,0 +1,19 @@
+### zookeeper(ZookeeperLocatorParameters)
+Creates a locator that requests a service from Zookeeper which returns a path (a string that is formatted within ```manager()```) to the service. Also checks the connection and emits an event based on the status of the client.
+##### Arguments
+* ```parameters``` -  Provides information of type ```ZookeeperLocatorParameters``` about the specified Zookeeper locator. ```ZookeeperLocatorParameters``` is an interface for the locator that describes the client of the ```servers```.
```

Comment:
This statemenet: ```ZookeeperLocatorParameters``` is an interface for the locator that describes the client of the ```servers``` is not correct. `ZookeeperLocatorParameters` is an interface for the argument to the locator, not the locator itself.